app/controllers/admin/versions_controller.rb

```diff
@@ -1,17 +1,23 @@
 module Admin
   class VersionsController < Admin::BaseController
-    skip_authorization_check
+    load_resource :conference, find_by: :short_title, only: :index
+    load_and_authorize_resource class: PaperTrail::Version
 
     def index
-      authorize! :index, PaperTrail::Version.new(item_type: 'User')
-      conf_ids_for_organizer = current_user.is_admin? ? Conference.pluck(:id) : Conference.with_role(:organizer, current_user).pluck(:id)
-      @versions = PaperTrail::Version.where(["conference_id IN (?) OR item_type = 'User'", conf_ids_for_organizer])
+      @conferences_with_role = []
+      @conferences_with_role = current_user.is_admin? ? Conference.pluck(:short_title) : Conference.with_role([:organizer, :cfp, :info_desk], current_user).pluck(:short_title)
```

Comment:
Organizations have multiple conferences. I can't find a way to get all conferences from a `ActiveRecord::Relation` of multiple organizations in a single line query using active record.
Do let me know if it's possible :thinking: